### PR TITLE
[Translate ru] Resources -> writing article

### PR DESCRIPTION
**Pull request by DmitrySkripunov**
[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/ru/review/406)

### CODE REVIEW

File: src/content/ru/resources/writing-an-article.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+Все документы должны иметь минимальный набор YAML front matter включающий ссылку
+на `_project.yaml`, и еще одну на `_book.yaml`.
+
+```
+project_path: /web/_project.yaml
+book_path: /web/section/_book.yaml
+```
+
+Note: If the path to the project or book cannot be found, the page will not
```
Comment (petele) [inline src/content/ru/resources/writing-an-article.md]: This should be translated, **except** for `Note:`

File: src/content/ru/resources/writing-an-article.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+<table class="responsive">
+  <tbody>
+    <tr>
+      <th colspan="2">Other Attributes</th>
+    </tr>
+    <tr>
+      <td><code>full_width: true</code></td>
+      <td>
+        Relinquishes control over the layout for the area below the site
```
Comment (petele) [inline src/content/ru/resources/writing-an-article.md]: This should be translated

File: src/content/ru/resources/writing-an-article.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+Вы можете добавить описание страницы в блоке YAML front matter, которое
+используется в `meta` описания страницы. Описание должно быть коротким (<450
+символов), и представлять собой лишь краткое описание страницы.
+
+```
+description: Lorem ipsum
+```
+
+Внимание: Не добавляйте блоки <code> (или `) поле описания.
```
Comment (petele) [inline src/content/ru/resources/writing-an-article.md]: `Внимание:` should not be translated, it should remain `Caution:`

Comment (petele) [inline src/content/ru/resources/writing-an-article.md]: `<code>` should be `&lt;code&gt;`

File: src/content/ru/resources/writing-an-article.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+Заголовок страницы определяется первым H1-подобным тэгом с классом
+`.page-title`.
+Например:
+
+<pre class="prettyprint">
+&num; Writing an Article {: .page-title }
+</pre>
+
+Внимание: Заголовки страниц не должны включать никакие markdown или HTML тэги.
```
Comment (petele) [inline src/content/ru/resources/writing-an-article.md]: `Внимание:` should not be translated, it should remain `Caution:`

File: src/content/ru/resources/writing-an-article.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+## Автор и переводчик
+
+Чтобы добавить автора и переводчика используйте:
+
+<pre class="prettyprint">
+{% include "web/_shared/contributors/petelepage.html" %}
```
Comment (petele) [inline src/content/ru/resources/writing-an-article.md]: The opening tag here needs to be encoded: `&#123;% `

**Comment by sotayamashita** [comment]
signed

**Comment by petele** [comment]
@sotayamashita & @chikathreesix - this has changes by @mt-gitlocalize, which doesn't seem to have signed the CLA. Can you investigate please?

**Comment by petele** [comment]
@DmitrySkripunov - thanks for the contributions! Can you make the requested changes please?

**Comment by DmitrySkripunov** [comment]
Hi, @petele ! Sorry, for a waiting! I made fixes, please, look at them.

**Comment by DmitrySkripunov** [comment]
@petele, @agektmr Could you look why this PR is not merged, please?

**Pull request merged**